### PR TITLE
Improve docs for publicize repo

### PR DESCRIPTION
**Pull request by elise-ng**
Ref oursky/likedao#160 

- Add project description
- Add architecture docs
- Add license and contribution docs
- Add server layers description

### CODE REVIEW

File: docs/Contributing.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Contributing
+
+Thank you for considering to contribute to our project.
+
+We recommend you to open an issue for discussion before introducing new features or submitting bug fixes. After discussing with the project team and implementing please open a pull request for code review.
+
+Please ensure that your changes would pass test cases, linters and format checks. Commit messages should be precise and should mention the related issue number.
```
Comment (rickmak) [inline docs/Contributing.md:7]: Provide some command, like `make ci`.

File: docs/Contributing.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+Since the project team work on a defined roadmap, there might be delays in code review of external pull requests.
```
Comment (rickmak) [inline docs/Contributing.md:9]: No need to be too defensive, can remove this...

**Review by rickmak — CHANGES_REQUESTED** [review]
Some reorganizing suggestion.

File: docs/Architecture.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+## Tech Stack
+
+| Module     | Tech              |
+| ---------- | ----------------- |
+| Web App    | React, Typescript |
+| Server     | Golang, GraphQL   |
+| Database   | PostgreSQL        |
+| Deployment | Kubernetes, Helm  |
+
```
[the remark this answers is not in the document]
Comment (chihimng) [reply docs/Architecture.md:13]: added

**Pull request merged**